debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/AlgorithmVersion.java

```diff
@@ -53,6 +53,7 @@
 	public static final String ITABLE_VERSION = "ALG_VM_ITABLE_VERSION";
 	public static final String BYTECODE_VERSION = "ALG_VM_BYTECODE_VERSION";
 	
+	public static final String GC_OBJECT_HEADER_VERSION = "ALG_GC_OBJECT_HEADER_VERSION";
```

Comment:
I don't think this change is needed.

Reply:
See above - this is used to determine if we can use the generic J9Object structure or not.